### ISSUE TITLE
relion viewer cl2d plot changes doesnt work

### ISSUE THREAD

**Issue by azazellochg** [scipion-em/scipion-em-relion#141]
```
Computing average changes in offset, angles, and class membership
Computing data for iteration; 001
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.8/tkinter/__init__.py", line 1883, in __call__
    return self.func(*args)
  File "/home/azazello/soft/scipion3/scipion-pyworkflow/pyworkflow/gui/form.py", line 1259, in _visualizeVar
    self.visualizeCallback(self.paramName)
  File "/home/azazello/soft/scipion3/scipion-pyworkflow/pyworkflow/viewer.py", line 266, in _visualizeParam
    views = self._getVisualizeDict()[paramName](paramName)
  File "/home/azazello/soft/scipion-plugins/scipion-em-relion/relion/viewers/viewers.py", line 496, in _showChanges
    mdIters.setValue(label, mdOptimiser.getValue(label), objId)
TypeError: Unrecognized type to create MDObject
```

**Comment by azazellochg** [comment]
This fails because md.RLN_OPTIMISER_ACCURACY_TRANS was replaced by md.RLN_OPTIMISER_ACCURACY_TRANS_ANGSTROM

**Comment by azazellochg** [comment]
Fixed in #167
